dbc2val/Dockerfile

```diff
@@ -30,7 +30,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 RUN /opt/venv/bin/python3 -m pip install --upgrade pip \
     && pip3 install --no-cache-dir -r requirements.txt
 
-RUN pip3 install wheel && pip3 install scons && pip3 install pyinstaller && pip3 install patchelf && pip3 install staticx
+RUN pip3 install wheel && pip3 install scons && pip3 install pyinstaller && pip3 install patchelf==0.17.0.0 && pip3 install staticx
```

Comment:
I verified that i could build the docker container locally with this fix. But I would like to have some clue somewhere on why we need to fix it to this particular version. In a comment in this file, or in the commit or in the PR. If it is a known issue, possibly link to a place describing it

So that if someone looks at this line in 1 year can find out why it was needed and check if it is safe to remove the fixed version (which I assume we would need at some time in the future)